### PR TITLE
Modified file churn_prediction_for_game_developers

### PR DESCRIPTION
**Pull request by sudarshan-SpringML**
Modified file churn_prediction_for_game_developers
- [X] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [X] Follow the style and grammar rules outlined in the above notebook template.
- [X] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [X] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [X] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [X] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

### CODE REVIEW

File: notebooks/official/workbench/gaming_churn_prediction/churn_prediction_for_game_developers.ipynb [hunk 1]
```diff
@@ -0,0 +1,7314 @@
+{
```
Comment (andrewferlitsch) [inline notebooks/official/workbench/gaming_churn_prediction/churn_prediction_for_game_developers.ipynb:1]: <p>move dataset after Objective</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/809/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Comment (andrewferlitsch) [inline notebooks/official/workbench/gaming_churn_prediction/churn_prediction_for_game_developers.ipynb:1]: <p>use the template version</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/809/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Comment (andrewferlitsch) [inline notebooks/official/workbench/gaming_churn_prediction/churn_prediction_for_game_developers.ipynb:1]: <p>Google Cloud Notebooks -&gt; Vertex AI Workbench Notebooks</p><p> </p><p>Combine the two text cells</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/809/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

**Comment by sudarshan-SpringML** [comment]
@andrewferlitsch all changes done

File: notebooks/official/workbench/gaming_churn_prediction/churn_prediction_for_game_developers.ipynb [hunk 2]
```diff
@@ -0,0 +1,6397 @@
...
```
Comment (andrewferlitsch) [inline notebooks/official/workbench/gaming_churn_prediction/churn_prediction_for_game_developers.ipynb:1]: <p>Move H1 title into links cell</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/809/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (andrewferlitsch) [inline notebooks/official/workbench/gaming_churn_prediction/churn_prediction_for_game_developers.ipynb:1]: <p>remove execution output</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/809/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

**Comment by andrewferlitsch** [comment]
A couple of review  comments added and still need to remove execution output

**Comment by sudarshan-SpringML** [comment]
changes done

**Review by andrewferlitsch — APPROVED** [review]

**Pull request merged**